kpops/components/streams_bootstrap/producer/producer_app.py

```diff
@@ -0,0 +1,63 @@
+from typing import NoReturn
+
+from pydantic import BaseConfig, Extra
+from typing_extensions import override
+
+from kpops.component_handlers.helm_wrapper.model import HelmRepoConfig
+from kpops.components.base_components.kafka_app import KafkaApp
+from kpops.components.base_components.models.to_section import (
+    OutputTopicTypes,
+    TopicConfig,
+)
+from kpops.components.streams_bootstrap.app_type import AppType
+from kpops.components.streams_bootstrap.producer.model import ProducerValues
+
+
+class ProducerApp(KafkaApp):
+    """
+    Producer component
+
+    This producer holds configuration to use as values for the streams bootstrap produce helm chart.
+    """
+
+    _type = "producer"
+    app: ProducerValues
+
+    class Config(BaseConfig):
+        extra = Extra.allow
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    @override
+    def apply_to_outputs(self, name: str, topic: TopicConfig) -> None:
+        match (topic.type):
```

Comment:
is it possible to remove the parentheses around the subject?

Reply:
It is!

Reply:
cool, I would prefer this slightly leaner syntax. what do you think?

Reply:
Yeah feels more elegant 🌟